src/styles/backupNotification.styl

```diff
@@ -4,3 +4,6 @@
     margin-left .5rem
     background rgba(255, 255, 255, .16)
     border-radius 8px
+
+.backup-circular-progress-background
+    color rgba(255, 255, 255, .16)
```

Comment:
@zatteo How does this component behave on a non inverted home? Like the one for GL?

Reply:
Verdict : bad. Will be handled in another PR.